rotkehlchen/tests/unit/test_loopring.py

```diff
@@ -20,3 +53,17 @@ def test_loopring_accountid_mapping(database):
 
     db.remove_accountid_mapping(addr1)
     assert db.get_accountid_mapping(addr1) is None
+
+
+def test_query_balances(temp_loopring, inquirer):  # pylint: disable=W0613
+    """Test the get account balances and check that all the conversions
+    are done correctly.
+    """
+    loopring = temp_loopring
+    patched_loopring = patch_loopring(loopring)
+
+    with patched_loopring:
+        result = loopring.get_account_balances(1)
```

Comment:
nitpick: try to make a habit to add the arguments as keyword arguments in most places.

Why? It helps the reader understand what it is. For example here I raised my eyebrow at `1`. Did not understand why. I expected an address. So had to go look at the code to see it's the account id.

If you write `loopring.get_account_balances(account_id=1)` it would be much clearer.

Ofc this is on a case by case basis, and not always important. Just noting it here so you are aware.